src/services/user-service.ts

```diff
@@ -959,6 +964,7 @@ export const getSkillMapRatings = async (skill_map_administration_id: number, us
     user_skill_map_ratings: userSkillMapRatings,
     skill_map_administration: skillMapAdministration,
     skill_map_result_status: skillMapResult.status,
+    comments: comment?.comments,
```

Comment:
I think you can use `skillMapResult.comments` here

Reply:
done :3